app/controllers/ems_common.rb

```diff
@@ -53,6 +53,68 @@ def show_ad_hoc_metrics
     drop_breadcrumb(:name => @ems.name + _(" (Ad hoc Metrics)"), :url => show_link(@ems))
   end
 
+  def show_topology
```

Comment:
You can remove this method and instead use the `MoreShowActions` mixin which has the `show_topology` method.